src/stories/atoms/Shape/Shape.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Shape } from 'stories/atoms/Shape/Shape';
+
+export default {
+  title: 'Design System/Atoms/Shape',
+  component: Shape,
+} as ComponentMeta<typeof Shape>;
+
+const Template: ComponentStory<typeof Shape> = (args) => <Shape {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
So I'm not sure how deeply you dug into this line, but this is takes advantage of `function.prototype.bind` which "create a new function that, when called, has its `this` keyword set to the provided value, with a given sequence of arguments preceding any provided when the new function is called."  

This is one of the weirder bits of `Storybook` syntax but also a pretty graceful solution to recreating the `Template` function with different arguments.